lldb/source/Plugins/Process/gdb-remote/GDBRemoteCommunicationServerLLGS.cpp

```diff
@@ -1676,7 +1681,11 @@ GDBRemoteCommunication::PacketResult
 GDBRemoteCommunicationServerLLGS::Handle_vCont_actions(
     StringExtractorGDBRemote &packet) {
   StreamString response;
-  response.Printf("vCont;c;C;s;S;t");
+  if (m_current_process && m_current_process->CanResume()) {
+    response.Printf("vCont;c;C;s;S;t");
+  } else {
+    response.Printf("vCont");
```

Comment:
What does this do if the process can't resume anyway?

https://sourceware.org/gdb/current/onlinedocs/gdb.html/Packets.html#vCont-packet

Or is this just a WIP implementation, and doing something rather than nothing meant you didn't have to change a bunch of other stuff.